vscode-dotnet-runtime-library/src/EventStream/LoggingObserver.ts

```diff
@@ -31,6 +31,6 @@ export class LoggingObserver implements ILoggingObserver {
     }
 
     private writeLine(line: string) {
-        this.log.concat(line);
+        this.log = this.log.concat(line);

```

Comment:
consider using `this.log.push(line)` instead - `concat` is a copying call, so this will generate a lot of additional garbage for the GC to manage. `push` modifies the underlying array and handles updating the size of the array if necessary, so copying is kept to a minimum.

Reply:
Nice catch, thank you :)